osu.Game/Screens/Menu/MainMenu.cs

```diff
@@ -69,12 +69,22 @@ public class MainMenu : OsuScreen
 
         private ExitConfirmOverlay exitConfirmOverlay;
 
+        private ParallaxContainer buttonsContainer;
+        private SongTicker songTicker;
+
         [BackgroundDependencyLoader(true)]
         private void load(DirectOverlay direct, SettingsOverlay settings, OsuConfigManager config, SessionStatics statics)
         {
             holdDelay = config.GetBindable<float>(OsuSetting.UIHoldActivationDelay);
             loginDisplayed = statics.GetBindable<bool>(Static.LoginOverlayDisplayed);
 
+            AddInternal(songTicker = new SongTicker
```

Comment:
I don't like how the children are added in splitted way, makes the code unnecessarily long, mabye add this to the `AddRangeInternal` below.

Reply:
No. It should be below `ExitConfirmOverlay`